sky/data/data_transfer.py

```diff
@@ -88,8 +80,31 @@ def s3_to_gcs(s3_bucket_name: str, gs_bucket_name: str) -> None:
         }
     }
 
-    result = storagetransfer.transferJobs().create(body=transfer_job).execute()
-    logger.info(f'AWS -> GCS Transfer Job: {json.dumps(result, indent=4)}')
+    response = storagetransfer.transferJobs().create(
+        body=transfer_job).execute()
+    operation = storagetransfer.transferJobs().run(jobName=response['name'],
+                                                   body={
+                                                       'projectId': project_id
+                                                   }).execute()
```

Comment:
Curious - are there any benefits other than readability to calling `transferJobs().create()` followed by `transferJobs().run()` instead of setting the `schedule` field to current time and only calling `transferJobs().create()` like we had before? 

(Just to be clear, I prefer the former as we have now, but curious if there's some other reason to do it this way)

Reply:
The main reason for using the current interactive way to start the job is that we can get the name of the `operation` for the `run().execute()` (this is different from the name of the submitted `TransferJob`, in the `response`). With the name, we don't have to list all the running transfering jobs and find out the correct operation name that is scheduled by the cloud in L98 below. Also, since we will only run the `TransferJob` once and in a blocking manner, I feel like having a `schedule` field in the specification can be a bit misleading.